sites/fast-website/src/public/index.html

```diff
@@ -34,8 +34,13 @@
         </header>
         <!-- header:end -->
 
+        <!-- side-navigation:start -->
+        <site-side-navigation category="links" class="social-links"></site-side-navigation>
+        <site-side-navigation category="scroll" sections="5" class="section-scroll"></site-side-navigation>
+        <!-- side-navigation:end -->
+
         <!-- hero-section:start -->
-        <section class="section">
+        <section class="section" id="section-0">
```

Comment:
Thank you for mentioning this. I will go through and update the names and functions associated so that it can run off of names instead of indexing through the numbers.

Reply:
I have made these changes. I have updated the link for the top section to be #top. I needed to have it still having an id to differentiate it from the other sections currently. I believe I should be able to remove the id as soon as I have applied the index updates.